flow/designs/gf12/tinyRocket/metadata-base-ok.json

```diff
@@ -31,91 +31,91 @@
   "cts__timing__drv__max_fanout__pre_repair": 0.0,
   "cts__timing__drv__max_cap__pre_repair": 0.0,
-  "cts__timing__setup__ws__pre_repair": -353.87,
-  "cts__timing__setup__tns__post_repair": -133510.73,
-  "cts__timing__setup__ws__post_repair": -353.87,
-  "cts__timing__drv__max_slew__post_repair": 8.0,
+  "cts__timing__setup__ws__pre_repair": -468.3,
+  "cts__timing__setup__tns__post_repair": -178210.95,
+  "cts__timing__setup__ws__post_repair": -468.3,
+  "cts__timing__drv__max_slew__post_repair": 43.0,
   "cts__timing__drv__max_fanout___post_repair": 0.0,
   "cts__timing__drv__max_cap__post_repair": 0.0,
-  "cts__timing__setup__tns": -1705.11,
-  "cts__timing__setup__ws": -19.83,
-  "cts__timing__drv__max_slew": 0.0,
+  "cts__timing__setup__tns": -27289.88,
+  "cts__timing__setup__ws": -90.16,
+  "cts__timing__drv__max_slew": 45.0,
   "cts__timing__drv__max_fanout": 0.0,
-  "cts__design__instance__count__hold_buffer": 260.0,
-  "globalroute__clock__latency__min": 125.81,
-  "globalroute__clock__latency__max": 190.05,
-  "globalroute__clock__skew__worst": 58.57,
-  "globalroute__timing__setup__tns": -3271.09,
-  "globalroute__timing__setup__ws": -91.71,
-  "globalroute__timing__drv__max_slew": 67.0,
+  "cts__design__instance__count__hold_buffer": 374.0,
+  "globalroute__clock__latency__min": 124.6,
+  "globalroute__clock__latency__max": 180.56,
+  "globalroute__clock__skew__worst": 50.69,
+  "globalroute__timing__setup__tns": -27602.57,
+  "globalroute__timing__setup__ws": -124.6,
+  "globalroute__timing__drv__max_slew": 38.0,
   "globalroute__timing__drv__max_fanout": 0.0,
   "globalroute__timing__drv__max_cap": 0.0,
-  "globalroute__timing__clock__slack": -91.713,
-  "globalroute__timing__clock__period": 847.128,
-  "globalroute__runtime__total": "0:29.64",
-  "globalroute__cpu__total": 27.71,
-  "globalroute__mem__peak": 816364.0,
-  "detailedroute__route__wirelength": 432921.0,
-  "detailedroute__route__vias": 247264.0,
+  "globalroute__timing__clock__slack": -124.603,
+  "globalroute__timing__clock__period": 878.373,
+  "globalroute__runtime__total": "0:19.64",
+  "globalroute__cpu__total": 19.04,
+  "globalroute__mem__peak": 844248.0,
+  "detailedroute__route__wirelength": 391382.0,
+  "detailedroute__route__vias": 245701.0,
   "detailedroute__route__drc_errors": 0,
-  "detailedroute__runtime__total": "5:50.92",
-  "detailedroute__cpu__total": 3040.99,
-  "detailedroute__mem__peak": 16800616.0,
-  "finish__clock__latency__min": 100.68,
-  "finish__clock__latency__max": 155.73,
-  "finish__clock__skew__worst": 50.43,
-  "finish__timing__setup__tns": 0.0,
-  "finish__timing__setup__ws": 36.47,
+  "detailedroute__runtime__total": "5:40.61",
+  "detailedroute__cpu__total": 2016.16,
+  "detailedroute__mem__peak": 16484684.0,
+  "finish__clock__latency__min": 101.52,
+  "finish__clock__latency__max": 150.9,
+  "finish__clock__skew__worst": 44.6,
+  "finish__timing__setup__tns": -5406.17,
+  "finish__timing__setup__ws": -66.96,
```

Comment:
Worse timing and 114 (43%) more buffers (`cts__design__instance__count__hold_buffer`)